docs/nix/unix_cheat_sheet.md

```diff
@@ -0,0 +1,35 @@
+| Command | Description |
+|---------|-------------|
+|**whoami**|who you are logged in as|
+|**pwd**|show current directory|
```

Comment:
would be better to describe as "print working directory," which it is the acronym for.